resources/authgear/templates/en/translation.json

```diff
@@ -110,7 +110,7 @@
   "copied-button-label": "Copied",
   "get-new-recovery-codes-button-label": "Get new codes",
   "download-button-label": "Download",
-  "disconnect-button-label": "Disconnect",
+  "remove-biometric-button-label": "Disconnect",
```

Comment:
Do not change v1 translation key.

Reply:
This commit was actually fixed up last week, just tried rebase and pushed again